prov/util/src/util_ep.c

```diff
@@ -87,7 +119,6 @@ int ofi_endpoint_init(struct fid_domain *domain, const struct util_prov *util_pr
 	ep->caps = info->caps;
 	ep->domain = util_domain;
 	ep->progress = progress;
-	ep->caps = info->caps;
```

Comment:
The statement is redundant (this is double set) the previous one is in 119 line of code